src/data/translation.ts

```diff
@@ -17,6 +17,12 @@ export enum TimeFormat {
   twenty_four = "24",
 }
 
+export enum NumericDateFormat {
+  language = "language",
+  MDY = "MDY",
+  DMY = "DMY",
```

Comment:
Why so little options? There are a lot more possibilities? Like I'm in the Netherlands, if I want the UI in English, but the dates in my normal format I would use `D-M-YY` not `DD/MM/YY`.

I would at least expect a system option here, like we have for the others